modules/ppcp-axo/src/Helper/PropertiesDictionary.php

```diff
@@ -0,0 +1,65 @@
+<?php
+/**
+ * Properties of the AXO module.
+ *
+ * @package WooCommerce\PayPalCommerce\Axo\Helper
+ */
+
+declare(strict_types=1);
+
+namespace WooCommerce\PayPalCommerce\Axo\Helper;
+
+/**
+ * Class PropertiesDictionary
+ */
+class PropertiesDictionary {
+
+	/**
+	 * Returns the list of possible email widget options.
+	 *
+	 * @return array
+	 */
+	public static function email_widget_options(): array {
+		return array(
+			'render'     => __( 'Render email input', 'woocommerce-paypal-payments' ),
+			'use_widget' => __( 'Use email widget', 'woocommerce-paypal-payments' ),
+		);
+	}
+
+	/**
+	 * Returns the list of possible address widget options.
+	 *
+	 * @return array
+	 */
+	public static function address_widget_options(): array {
+		return array(
+			'render'     => __( 'Render address options list', 'woocommerce-paypal-payments' ),
+			'use_widget' => __( 'Use address widget', 'woocommerce-paypal-payments' ),
+		);
+	}
+
+	/**
+	 * Returns the list of possible address widget options.
+	 *
+	 * @return array
+	 */
+	public static function payment_widget_options(): array {
+		return array(
+			'render'     => __( 'Render payment options list', 'woocommerce-paypal-payments' ),
+			'use_widget' => __( 'Use payment widget', 'woocommerce-paypal-payments' ),
+		);
+	}
```

Comment:
These 3 methods seems not used in the codebase, if this is intended leave them otherwise delete, thanks.